src/Form/index.tsx

```diff
@@ -934,9 +935,12 @@ function Form({
         // @ts-expect-error TS(2345): Argument of type 'Promise<any[]>' is not assignabl... Remove this comment to see the full error message
         .fetchSession(formPromise, true)
         .then(([session, steps]) => {
-          if (!session) {
+          if (!session || session.collaborator?.invalid) {
             setFormSettings({ ...formSettings, formOff: true });
             return;
+          } else if (session.collaborator?.completed) {
+            setCollaboratorCompleted(true);
```

Comment:
instead of creating a new collaboratorcompleted state variable, i wonder if we can make **formOff** an enum instead of a boolean instead